backend/core/settings_schemas.py

```
@@ -0,0 +1,283 @@
+            data = json.loads(json_str)
+            return cls.from_dict(data)
+        except (json.JSONDecodeError, TypeError, ValueError) as e:
+            logger.warning(f"Failed to parse feature flags from JSON: {e}")
+            return cls()  # Return defaults on error
+
+
+@dataclass
+class SystemSettings:
+    """Unified container for all DB-driven runtime settings."""
+
+    followup: "FollowUpSettings" = field(default_factory=lambda: None)  # Import from config.py
```

Ruff (0.12.2)

Undefined name `FollowUpSettings`

(F821)